lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -54,24 +55,40 @@ my @availableLocalCurrencies = (
     'eur',
     'jpy',
     'rur',
+    'rub',
     'uah'
 );
 
+foreach my $currency (@currencies){
+    chomp($currency);
+    my @currency = split(/,/,$currency);
+
+    if(!grep(/^$currency[0]$/, @availableLocalCurrencies) || !grep(/^$currency[0]$/, @excludedCurrencies)) {
+        push(@cryptoTriggers, @currency);
+    }
+
+    push(@currTriggers, @currency);
+    $currHash{$currency[0]} = \@currency;
+}
+
 my %excludedCurrencies = map { $_ => 1 } @excludedCurrencies;
 my %topCurrencies = map { $_ => 1 } @topCurrencies;
 my %availableLocalCurrencies = map {$_ => 1} @availableLocalCurrencies;
 
 #Define regexes
 my $currency_qr = join('|', @currTriggers);
-my $question_prefix = qr/(?:convert|what (?:is|are|does)|how (?:much|many) (?:is|are))?\s?/;
-my $rate_qr = qr/\s(?:rate|exchange|exchange rate|conversion|price)/i;
+my $crypto_qr = join('|', @cryptoTriggers);
+my $question_prefix = qr/(?:convert|current value (?:of)?|what (?:is|are|does)|(?:conversion|exchange|fx) rate\s?(?:for)?|how (?:much|many) (?:is|are))?\s?/;
+my $rate_qr = qr/\s(?:rate|exchange|(?:exchange|fx) rate|conver(?:sion|ter)|calc(?:ulator)?|price)/i;
 my $into_qr = qr/\s(?:en|in|to|in ?to|to|from)\s/i;
 my $vs_qr = qr/\sv(?:ersu|)s\.?\s/i;
 my $number_re = number_style_regex();
 
+my $crypto_triggers = qr/^(?:$crypto_qr)(?:\s$rate_qr)?$/i;
 my $guard = qr/^$question_prefix($number_re*?\s+|)($currency_qr)(?:s)?(?:$into_qr|$vs_qr|$rate_qr|\s)?($number_re*?\s+|)($currency_qr)?(?:s)?\??$/i;
+my $generic_triggers = qr/^(?:((?:crypto\s?(currency)?)\s?(?:calc(?:ulator)?|conver(?:sion|ter)|exchanges?)?)|(?:coin exchanges?))$/;
 
-triggers query_lc => qr/$currency_qr/;
+triggers query_lc => qr/$generic_triggers|$currency_qr|$crypto_triggers/;
```

Comment:
fwiw, you could specify these separately if you wanted:
```perl
triggers query_lc => qr/$generic_triggers/;
triggers query_lc => qr/$currency_qr/;
triggers query_lc => qr/$crypto_triggers/;
```

Would be useful if they're anchored differently.